devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -163,6 +166,7 @@ public void execute(Task test) {
                     Task testResourcesTask = tasks.getByName(JavaPlugin.PROCESS_TEST_RESOURCES_TASK_NAME);
                     quarkusDev.dependsOn(classesTask, resourcesTask, testClassesTask, testResourcesTask, quarkusGenerateCode);
                     quarkusRemoteDev.dependsOn(classesTask, resourcesTask);
+                    quarkusTest.dependsOn(classesTask, resourcesTask, testClassesTask, testResourcesTask, quarkusGenerateCode);
```

Comment:
I'm not entirely sure, but maybe also `quarkusGenerateCodeTests`?

(Which should maybe be there for `quarkusDev` as well? As I said, not sure.)

Reply:
Maybe also CC @michalszynkiewicz

Reply:
👍🏻 I think test code generation should also be among the dependencies